src/eval.c

```diff
@@ -442,11 +439,9 @@ sds luaCreateFunction(client *c, robj *body) {
 
     /* Note that in case of a shebang line we skip it but keep the line feed to conserve the user's line numbers */
     if (luaL_loadbuffer(lctx.lua,(char*)body->ptr + shebang_len,sdslen(body->ptr) - shebang_len,"@user_script")) {
-        if (c != NULL) {
```

Comment:
Otherwise, we need to add the same check above

```
    if (evalExtractShebangFlags(body->ptr, &script_flags, &shebang_len, &err) == C_ERR) {
        addReplyErrorSds(c, err);
        return NULL;
    }
```